CMakeLists.txt

```diff
@@ -139,6 +139,14 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
 #    set(DEFAULT_MALLOC "jemalloc")
 endif()
 
+hpx_option(HPX_WITH_THREAD_STACKOVERFLOW_DETECTION
+  BOOL
+  "Enable stackoverflow detection for HPX threads/coroutines. (default: ON)"
+  ON ADVANCED)
```

Comment:
@sithhell: we were not able to resolve a similar configuration issue a while back (see: #2365). How would you suggest to do this (if you had a new idea we could go back and fix #2365 as well...)?